eslint.js

```diff
@@ -1,20 +1,44 @@
 'use strict';
 
-var config = {
+const config = {
```

Comment:
Guessing this doesn't matter that it's const since we don't lint against this file?

Reply:
Ooops, I missed this.  Yes it's because this repository doesn't have any test.  I could set `npm test` to run `eslint .` and put `module.exports = require('./eslint.js');` in `.eslintrc.js`?

Reply:
If that works, I wasn't sure if there was some weird recursive problem with using the config to lint itself 🤣 

This would also help a bunch with #16 because we could just lint a bunch of test files!

Reply:
Regarding #16 I think we need to look to other eslint configs / plugins to inspiration on testing.  Almost more important to verify that some forbidden stuff gives errors than it is to verify acceptable stuff does not error.

So this is updated, I have not enabled CI yet as I think we need to actually solve #16 for it to be worth running CI.

Reply:
Makes sense. I'll get this merged and released.